packages/dds/tree/src/tableSchema.ts

```diff
@@ -124,54 +141,59 @@ export namespace TableSchema {
 		readonly id: string;
 
 		/**
-		 * Gets the cell in the specified column
+		 * Gets the cell in the specified column, denoted by column ID.
 		 * @returns The cell if it exists, otherwise undefined.
-		 * @privateRemarks TODO: add overload that takes column ID.
 		 */
-		getCell(
-			column: TreeNodeFromImplicitAllowedTypes<TColumnSchema>,
-		): TreeNodeFromImplicitAllowedTypes<TCellSchema> | undefined;
+		getCell(columnId: string): TreeNodeFromImplicitAllowedTypes<TCellSchema> | undefined;
 
 		/**
-		 * Sets the cell in the specified column.
+		 * Sets the cell in the specified column, denoted by column ID.
 		 * @remarks To remove a cell, call {@link TableSchema.IRow.removeCell} instead.
-		 * @privateRemarks TODO: add overload that takes column ID.
 		 */
 		setCell(
-			column: TreeNodeFromImplicitAllowedTypes<TColumnSchema>,
+			columnId: string,
 			value: InsertableTreeNodeFromImplicitAllowedTypes<TCellSchema>,
 		): void;
 
 		/**
-		 * Removes the cell in the specified column.
-		 * @privateRemarks TODO: add overload that takes column ID.
+		 * Removes the cell in the specified column, denoted by column ID.
 		 */
-		removeCell(column: TreeNodeFromImplicitAllowedTypes<TColumnSchema>): void;
+		removeCell(columnId: string): void;
```

Comment:
Maybe not for this PR, but I think this API should return the removed cell.

Reply:
I'll save it for a future PR, but leave a TODO now.
```suggestion
		// TODO: return removed cell
		removeCell(columnId: string): void;
```